gozo-stores/src/components/HorizontalCard.jsx

```diff
@@ -9,14 +15,33 @@ export function HorizontalCard(props) {
 
             <div class="product-card-body">
                 <div class="product-name"> {props.name} </div>
-                <div class="product-price"> <span class="final-price"></span> {props.finalPrice} <span class="og-price text-s"><strike> {props.ogPrice} </strike></span></div>
+
+                <div class="product-price"> 
+                    <span class="final-price"></span> {props.finalPrice} INR <span class="og-price text-s"><strike> {props.ogPrice} INR </strike></span>
+                </div>
+
                 <div class="product-offer"> {props.discount} </div>
+
                 <div class="product-quantity">Quantity: <span>
-                    <button>-</button> <span>{props.quantity}</span> <button>+</button>
+                    <button onClick={() => {
+
+                        //if the quantity is 1 then removes the product
+                        //instead of showing quantity in 0 or negative number
+                        if(props.quantity === 1) {
+                            deleteFromCart(token, props.productId)
+                        }
+                        else {
+                            updateCartQuantity(token, props.productId, "decrement")}
+                        }
+                    }>-</button> 
+
+                    <span>{props.quantity}</span> 
+
+                    <button onClick={() => updateCartQuantity(token, props.productId, "increment")}>+</button>
```

Comment:
For good practice you can pass the payload with uppercase letter it's just a suggestion not mandatory.